docs/content.zh/docs/connectors/flink-sources/mysql-cdc.md

```diff
@@ -433,6 +446,13 @@ Flink SQL> SELECT * FROM orders;
       <td>TIMESTAMP_LTZ(3) NOT NULL</td>
       <td>当前记录表在数据库中更新的时间。 <br>如果从表的快照而不是 binlog 读取记录，该值将始终为0。</td>
     </tr>
+    <tr>
+      <td>row_kind</td>
```

Comment:
Please update the example to add `operation     STRING METADATA FROM 'row_kind' VIRTUAL,` too.